base/set.jl

```diff
@@ -17,6 +17,8 @@ Set() = Set{Any}()
 Construct a [`Set`](@ref) of the values generated by the given iterable object, or an
 empty set. Should be used instead of [`BitSet`](@ref) for sparse integer sets, or
 for sets of arbitrary objects.
+!!! note
+    `Set` is not ordered when iterating over.
```

Comment:
Something about the grammar seems off here. Perhaps "Iteration over `Set`s is unordered"?

Reply:
Maybe ``The iteration order of a `Set` should not be relied upon.``? I probably wouldn't say that the iteration itself is unordered.